### PR TITLE
Replace Bintray by Github Packages

### PR DESCRIPTION
**Pull request by maxdec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ A Micronaut library to lock method executions in a distributed environment
...
 
-Add our bintray repository to your pom.xml or build.gradle
+Add our Github Packages repository to your pom.xml or build.gradle
 
 build.gradle
 ```
 repositories {
     maven {
-        url  "https://dl.bintray.com/uberall/micronaut" 
+        url  "https://maven.pkg.github.com/uberall/micronaut-distributed-lock"
```
Comment (maxdec) [inline README.md:14]: Not 100% sure about that one

**Comment by peh** [comment]
Nice, but why?

**Comment by maxdec** [comment]
Because of the Bintray sunset? https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

**Comment by peh** [comment]
wow! Didn't know!

File: gradle.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 micronautVersion=2.0.0
 libVersion=1.2.1
-testcontainersVersion=1.15.0-rc2
+testcontainersVersion=1.15.2
```
Comment (maxdec) [inline gradle.properties:3]: This is to get this fix: https://github.com/testcontainers/testcontainers-java/issues/3574

**Pull request merged**